modules/mesh-layers/src/scenegraph-layer/scenegraph-layer-vertex.glsl.js

```diff
@@ -1,36 +1,98 @@
 export default `\
+#if (__VERSION__ < 300)
+  #define _attribute attribute
```

Comment:
We probably discussed this already, but I thought we already injected a prologue with these defines.

Reply:
No we are not. **BUT** for modules (not the main code) we handle this with regex.

Reply:
Just checked. it is not in the prologue, it is in the transpiler. Seems we still don't transpile modules.

Reply:
We transpile  the modules but not the main code.